### PR TITLE
Update SleepIQ with config flow instructions

### PR DESCRIPTION
**Pull request by mfugate1**
## Proposed change

Update SleepIQ documentation with new details about how to configure the integration with the config flow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/64850
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sleepiq.markdown [hunk 1]
```diff
@@ -13,21 +13,26 @@ ha_platforms:
...
-
-You will need an account on [SleepIQ](https://sleepiq.sleepnumber.com/) to use this component.
+You will need an account on [SleepIQ](https://sleepiq.sleepnumber.com/) to use this integration.
 
 ## Configuration
 
-To set it up, add the following information to your `configuration.yaml` file:
+1. In the **Configuration** > **Devices & Services** menu, click **+** and then select "SleepIQ" from the pop-up menu.
+2. In the pop-up box, enter your username and password for your SleepIQ account. Optionally, choose a custom scan interval (the default is to check for updates every 60 seconds).
+3. Click **Submit**. If the account information is valid, a configuration entry will be created and your SleepNumber sensors will be available in Home Assistant.
```
Comment (DurgNomis-drol) [inline source/_integrations/sleepiq.markdown]: Below code will add a big button, making it possible for the user to open the config flow on their Home Assistant instance directly.

```
{% include integrations/config_flow.md %}
```

Comment (mfugate1) [reply source/_integrations/sleepiq.markdown]: Done, swapped out the manual steps for the button

File: source/_integrations/sleepiq.markdown [hunk 2]
```diff
@@ -13,21 +13,22 @@ ha_platforms:
...
+The SleepIQ integration lets you view sensor data from [SleepIQ by SleepNumber](https://www.sleepnumber.com/sleepiq-sleep-tracker). In particular, it lets you see the occupancy and current SleepNumber (ie current firmness) of each side of a SleepNumber bed.
 
-## Setup
+You will need an account on [SleepIQ](https://sleepiq.sleepnumber.com/) to use this integration.
 
-You will need an account on [SleepIQ](https://sleepiq.sleepnumber.com/) to use this component.
+{% include integrations/config_flow.md %}
 
-## Configuration
+## Manual Configuration
```
Comment (frenck) [inline source/_integrations/sleepiq.markdown]: Once the configuration flow is in, existing entries should be migrated and configuration via YAML be deprecated.

Therefore, this section should be removed, as one should not be using it.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/sleepiq.markdown (hunk 2 above)
Comment (mfugate1) [reply source/_integrations/sleepiq.markdown]: Done

**Review by frenck — APPROVED** [review]
Thanks, @mfugate1 👍

**Pull request merged**